RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplateSplit.cc

```diff
@@ -103,7 +103,7 @@ int SiPixelTemplateSplit::PixelTempSplit(int id, float cotalpha, float cotbeta,
 {
     // Local variables 
 	int i, j, k, binq, midpix, fypix, nypix, lypix, logypx, lparm;
-	int fxpix, nxpix, lxpix, logxpx, shifty, shiftx, nyzero[TYSIZE];
+	int fxpix, nxpix, lxpix, logxpx, shifty, shiftx, nyzero[TYSIZE]{};
```

Comment:
The old initialization on line 211 is not needed if the array is initialized when it is defined. The definition should be moved to line 226 because the array isn't actually used before that. We only want to allocate and initialize the array if it is really needed.